doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-RPi/index.md

```diff
@@ -5,4 +5,46 @@ permalink: /:path/
 sort_idx: 60
 ---
 
-SDK integration instructions for select platforms
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The guide provides information on how to **cross-compile** C SDK for Raspberry PI.
+Alternatively, you can build the Kaa C endpoint SDK right on the Raspberry Pi board.
+Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/) for further details.
+
+# Install dependecies
+
+**The further instructions must be executed on the host machine.**
+
+**Verified against:**
+
+ - **Host OS:** Ubuntu 14.04 LTS 32-bit LTS
+ - **Target OS:** Raspbian Jessie (2015-11-21)
+
+1. Download and install toolchain.
```

Comment:
Dependencies are not installed. (`sudo apt-get install build-essentials cmake ...`.)